examples/preprocessing/plot_run_ica.py

```diff
@@ -23,11 +23,8 @@
 
 ###############################################################################
 # Read and preprocess the data. Preprocessing consists of:
-#
```

Comment:
Turns out that this line was necessary

https://7490-1301584-gh.circle-artifacts.com/0/html/auto_examples/preprocessing/plot_run_ica.html

Reply:
(the ones in between the `-` lines shouldn't be, though)

FYI I also did a merge commit to resolve a conflict before noticing this